src/pages/overlay/components/overlay/Overlay.tsx

```diff
@@ -59,7 +59,10 @@ const overlayOptions = [
   },
 ] as const;
 
-type OverlayKey = (typeof overlayOptions)[number]["key"];
+export const isValidOverlayKey = (key: string) =>
+  key == "" || overlayOptions.some((option) => option.key === key);
```

Comment:
```suggestion
  key === "" || overlayOptions.some((option) => option.key === key);
```
Any reason we can't use strict equality here?